dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -869,3 +905,11 @@ internal open class RumViewScope(
         }
     }
 }
+
+private fun VitalInfo.toPerformanceMetric(): ViewEvent.FlutterBuildTime {
```

Comment:
Having one specific type that is `FlutterBuildTime` might be harder to change later. Maybe we can create a dedicated type for this to be more generic?

Reply:
This came from the generator, I think it just chose the first property that used the type? @0xnm do we maybe want to adjust the generator somehow?

Reply:
I'm not quite sure what is the concern here. Can you please elaborate?

Reply:
Basically just that "FlutterBuildTime" is used as the type for things that are not FlutterBuildTime. But that type came from the schema generator so I just rolled with it.

Reply:
Well, this is how it is called in the schema. If it needs to have more generic name, the property should have generic name as well. But I don't see the issue here, because once we add a property for JS, it will get its own type. Types are basically not reused.

Reply:
In the generator we have a way to reuse types when they have the same schema. I'll create a task to try and find a better way to name types